task/push-dockerfile-oci-ta/0.1/push-dockerfile-oci-ta.yaml

```diff
@@ -45,10 +54,21 @@ spec:
     - name: IMAGE_REF
       description: Digest-pinned image reference to the Dockerfile image.
   volumes:
+    - name: trusted-ca
```

Comment:
How come we don't need to add `useTAVolumeMount` to the recipe to have this?

Reply:
@yftacherzog, thats because it was added to the non ta task and was generated from it

Reply:
That was also the case when we did all previous tasks.
We originally added the support for `useTAVolumeMount` to allow that code to add that volume mount to the TA task (in those cases, the non-oci-ta also had that volume mount).

Reply:
Talking to @avi-biton, I understand now that it's not needed here as the mount was added as part of the step template.